core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -56,30 +56,36 @@ void setProperty(final String property, final String value)
    * property overrides in ZooKeeper. Only properties which can be stored in ZooKeeper will be
    * accepted.
    *
-   * <P>
+   * <p>
+   * Accumulo has multiple layers of properties that for many APIs and SPIs presented as single
```

Comment:
... APIs and SPIs **are** presented as **a** single merged view.

Reply:
fixed in 767ba52a6192d70ffa8b2152f12127cb7c560c42